.changeset/five-gifts-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"@snapshot-labs/sx": patch
+---
+
+fix issue with decimal voting power for offchain proposals
```

Comment:
```suggestion
change default decimals for remote-vp strategy to 18
```

Current message doesn't really make sense on its own from perspective of external users of sx.js